package/raphael.js

```diff
@@ -5420,7 +5420,7 @@
         l = 0;
         for (; l < animationElements.length; l++) {
             var e = animationElements[l];
-            if (e.el.removed || e.paused) {
+            if (e.el.removed || e.paused || e.parentEl && e.parentEl.e.paused) {
```

Comment:
This checks immediate parent. Just FYI this will not work with parent higher than two levels in paused state. However, a recursion here will have performance hit.

One solution is to have a hash of paused elements and lookup from there and ensure that the hash is cleared on resume or remove to avoid memory leaks.